.github/workflows/publish-charm.yaml

```diff
@@ -34,6 +34,11 @@ jobs:
           credentials: "${{ secrets.CHARMCRAFT_AUTH }}"
           github-token: "${{ secrets.GITHUB_TOKEN }}"
           channel: latest/edge
+      - name: Publish libs
+        env:
+          CHARMCRAFT_AUTH: "${{ secrets.CHARMCRAFT_AUTH }}"
+        run: |
+          charmcraft publish-lib charms.vault_k8s.v0.vault_kv
```

Comment:
There's also a charming action to release libraries: https://github.com/canonical/charming-actions/tree/main/release-libraries

Reply:
Thanks, I changed the workflow to use it.